zio-http/src/main/scala/zio/http/ClientSSLConfig.scala

```diff
@@ -0,0 +1,11 @@
+package zio.http
+
+sealed trait ClientSSLConfig
+
+object ClientSSLConfig {
+  case object Default                                                                         extends ClientSSLConfig
```

Comment:
There's overlap with the server one--does it make sense to unify them, or not?

Reply:
There's overlap, but I don't think it's worth unifying them, because then we'd have to prefix the branches like: `ClientFromResource`, or have something like `case class SSLConfig(..., ..., clientOrServer: ClientOrServer)`